core/android/ui/previews/src/main/java/com/vgleadsheets/composables/previews/screens/details/ComposerDetail.kt

```diff
@@ -73,8 +74,8 @@ private fun composerScreenLoadingState(): State {
 
     val screenState = State(
         composer = LCE.Loading(stringGenerator.generateName()),
-        games = LCE.Loading(stringGenerator.generateName()),
         songs = LCE.Loading(stringGenerator.generateName()),
+        games = LCE.Loading(stringGenerator.generateName()),
```

Comment:
looks like this may have been accidentally moved?

Reply:
moved back